web/cobrands/highways/assets.js

```diff
@@ -55,9 +55,6 @@ fixmystreet.assets.add(defaults, {
     actions: {
         found: function(layer, feature) {
             var category = fixmystreet.reporting.selectedCategory().category;
-            if (category.match('NH')) {
```

Comment:
I could not work out what this code was for, or why you would want to deselect when a NH category was being picked.